airflow/providers/amazon/aws/hooks/base_aws.py

```diff
@@ -820,7 +826,7 @@ def get_waiter(
         self,
         waiter_name: str,
         parameters: dict[str, str] | None = None,
-        deferrable: bool = False,
+        deferrable: bool = DEFAULT_DEFERRABLE,
```

Comment:
Similar to my other comment, this should also stay as `False`. When a Trigger uses an async waiter, it will set the `deferrable` parameter as `True` (see [redshift_cluster](https://github.com/apache/airflow/blob/main/airflow/providers/amazon/aws/triggers/redshift_cluster.py#L124))

Reply:
Thank you for reminding me! Just updated it!

Reply:
In that case, we probably should change the ones in `airflow/providers/amazon/aws/hooks/base_aws.py`, `airflow/providers/amazon/aws/hooks/batch_waiters.py` and `airflow/providers/amazon/aws/waiters/base_waiter.py` as well?

Reply:
I tried to apply this change to aws hooks and waiters

Reply:
yes that's right. The way deferrables are implemented, `async_conn` is the only place (internally) that needs to set `deferrable` to `True`. At the operator level, the user can decide what they want for the `deferrable` param.

Reply:
Thanks for confirmation!